### PR TITLE
Improve keyboard accessibility

### PR DESCRIPTION
**Pull request by AlexGStapleton**
This is the WB version of https://github.com/siteorigin/siteorigin-panels/pull/772.

### CODE REVIEW

**Comment by AlexGStapleton** [comment]
@Misplon I haven't spotted any other inaccessible widgets/areas. Let me know if you find any.

File: base/js/admin.js [hunk 1]
```diff
@@ -295,8 +298,10 @@ var sowbForms = window.sowbForms || {};
 						}
 					} );
 			};
-			$fields.filter('.siteorigin-widget-field-type-widget, .siteorigin-widget-field-type-section').find('> label').click(expandContainer);
-			$fields.filter('.siteorigin-widget-field-type-posts').find('.posts-container-label-wrapper').click(expandContainer);
+			$fields.filter('.siteorigin-widget-field-type-widget, .siteorigin-widget-field-type-section').find('> label')
+			.on( 'click keyup', expandContainer)
```
Comment (Misplon) [inline base/js/admin.js:302]: A space is needed after `expandContainer`.

File: base/js/admin.js [hunk 2]
```diff
@@ -295,8 +298,10 @@ var sowbForms = window.sowbForms || {};
...
+			.attr( 'tabinex', 0 );
+			$fields.filter('.siteorigin-widget-field-type-posts').find('.posts-container-label-wrapper').on( 'click keyup', expandContainer);
```
Comment (Misplon) [inline base/js/admin.js:304]: A space is needed after `expandContainer`.

**Review by Misplon — APPROVED** [review]

**Comment by Misplon** [comment]
@AlexGStapleton Can you take a quick look at the above conflicts when able? Thanks, mate.

**Review by gregpriday — APPROVED** [review]

**Pull request merged**